packages/cli/src/index.ts

```diff
@@ -93,11 +93,14 @@ async function execCmd(args: string[]) {
   await Promise.all(
     packages.map((pkg) => {
       return execLimit(async () => {
-        let { code } = await spawn(args[0], args.slice(1), {
-          cwd: pkg.dir,
-          stdio: "inherit",
+        const proc = exec(args[0], args.slice(1), {
```

Comment:
Could u refactor the touched lines to this style (when possible)?
```suggestion
        const { exitCode } = await exec(args[0], args.slice(1), {
```

or at least to this if the `.exitCode!` is problematic:
```suggestion
        const result = await exec(args[0], args.slice(1), {
```

Reply:
Unfortunately, the `exec()` constructor and the return type of the `await proc` are different types. 

`const proc = exec()` is an object that holds a "prepared execution" that you can attach streams/events/whatever on. It is also .then()-able, but that returns a different object...
`const result = await proc` is a Result type that only has `stdout` and `stderr`. It doesn't have `exitCode` (though I think it really should, there is an issue upstream, will see if I can open a PR for it).

The `proc.exictCode!` should be correct, because `exitCode` should always be set if we have awaited the `proc`.

As weird as the code looks right now, it is the only way to write it as far as I can see.

Reply:
Let's wait a little bit then before landing this. It would be great if `exitCode` thing would be improved upstream.

Reply:
Oh, I was holding it wrong!

When the process has an non-zero exit code, an exception is thrown.

This means that if we don't read stdout/stderr, we can just do:
```
    await exec("yarn", args.slice(1), {
      nodeOptions: {
        cwd: matchingPackages[0].dir,
        stdio: "inherit",
      },
    });
 ```
 
 And we can completely remove the `ExitError` class, as `tinyexec` throws its own `NonZeroExitError`

Reply:
Merged the upstream change in `tinyexec`: https://github.com/tinylibs/tinyexec/pull/38

The PR is now good for re-review.